web/modules/custom/dpl_paragraphs/dpl_paragraphs.module

```diff
@@ -377,3 +377,32 @@ function dpl_paragraphs_field_widget_single_element_paragraphs_form_alter(array
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_paragraph__TYPE().
+ *
+ * Prepares variables for the Opening Hours app in the Opening Hours paragraph.
+ *
+ * @param array &$variables
+ *   Variables for the paragraph template, including the 'paragraph' entity.
+ */
+function dpl_paragraphs_preprocess_paragraph__opening_hours(&$variables): void {
```

Comment:
Consider moving this implementation to the `dpl_opening_hours` module.

Since this deals with a paragraph specifically targeted for the domain managed by this module I think it makes sense to keep all the related code there.

Reply:
Sure, this makes sense. 

Added a `dpl_opening_hours.module` and moved the preprocess there 👍